onelogin_aws_cli/configuration.py

```diff
@@ -0,0 +1,90 @@
+"""
+Static User Configuration models
+"""
+import configparser
+
+from onelogin_aws_cli.userquery import user_choice
+
+
```

Comment:
There are two reasons to abstract this out:

- encapsulation of the config class where it is aware of it's own file path, makes it easier to mock the file pointer for testing
 - Allow the definition of a `defaults` section in the config file to allow values in `[defaults]` to be applied to all sections unless over-ridden.
 - reduce the size of the `onelogin_aws_cli.OneloginAWS` class and